### DISCUSSION TITLE
How to make NFT plots?

### DISCUSSION THREAD

**Discussion by nicezic** [madMAx43v3r/chia-plotter#767] [Q&A]
I updated a new Madmax version, which supports NFT plots.
Do I need use '--contract arg'?  

If I create a plot with the farmer key and pool key as same as the previous version, is the plot NFT or OG plot?

Please, let me know how to self NFT plot with Madmax.

Thank you.

**Comment by gryan315** [comment]
Using the -p argument will make an OG plot. You need to create a plotnft in the chia software, then input the P2 singleton address for the -c argument when running mad max.

**Reply by altendky** [reply]
https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide

Step 4 refers to the Chia Network plotter, but the relevant options are the same.
